pkg/scheduler/plugins/coscheduling/util/gang_helper.go

```diff
@@ -56,11 +57,24 @@ func IsPodNeedGang(pod *v1.Pod) bool {
 	return GetGangNameByPod(pod) != ""
 }
 
-func ParsePgTimeoutSeconds(timeoutSeconds int32) (time.Duration, error) {
-	if timeoutSeconds <= 0 {
-		return 0, fmt.Errorf("podGroup timeout value is illegal,timeout Value:%v", timeoutSeconds)
+// GetWaitTimeDuration returns a wait timeout based on the following precedences:
+// 1. spec.scheduleTimeoutSeconds of the given pg, if specified
+// 2. given defaultTimeout, if not nil
+// 3. fall back to 0
+func GetWaitTimeDuration(pg *v1alpha1.PodGroup, defaultTimeout *metav1.Duration) time.Duration {
```

Comment:
`defaultTimeout` can be defined as value type and the defaultTimeout in args can't be nil since the default setter function will be called when scheduler init.

Reply:
Done, the defaultTimeout is 600s.